docs/ui/components/SDKTables/sdk-versions.json

```
@@ -20,7 +21,8 @@
       "xcode": "16.0+",
       "react-native": "0.79",
       "react-native-web": "0.20.0",
-      "react": "19.0.0"
+      "react": "19.0.0",
+      "node": "22.15.0"
```

```suggestion
      "node": "22.15.x"
```

maybe .x for each of these, so we don't have to track the patch versions?